obulaworld/mydiary_react
# #162321806 Setup jest and enzyme

#### What does this PR do?
 - Setup jest and enzyme

#### Description of Task to be completed?
- setup jest and enzyme chore
- run an example test

#### How should this be manually tested?
- start the app by running ```yarn start: dev```
the app should open on port ```8082``` on your browser

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#162321806](https://www.pivotaltracker.com/story/show/162321806, "Setup jest and enzyme")

#### Screenshots (if appropriate)


#### Questions:
